buildconfig/CMake/UiToPy.cmake

```diff
@@ -17,7 +17,7 @@ function(UiToPy5 ui_files target_name)
     # Command to run the translation
     add_custom_command(
       OUTPUT ${generated_file}
-      COMMAND ${py_exec} ${py_uic_py} ${source_file} -o ${generated_file}
```

Comment:
We can probably get rid of the `${py_uic_py}` variable now 👍

Reply:
I've purged a couple of unnecessary variables from that file